docs/helpshift-widget/api.mdx

```diff
@@ -0,0 +1,433 @@
+HelpshiftWidget("open");
+```
+
+## Close Widget {#close-widget}
+
+Similarly, for closing the widget window at any time, you can call the following API.
+
+```javascript
+HelpshiftWidget("close");
+```
+
+## Hide Widget {#hide-widget}
+
+You can hide the Helpshift widget (both the launcher and the widget window), by calling the following API.
+
+```javascript
+HelpshiftWidget("hide");
+```
+
+<Admonition type="info" title="Note">
+
+You may want to use this API when you do not want to show the widget on specific pages or to specific users.
+
+</Admonition>
+
+## Show Widget {#show-widget}
+
+You can have Helpshift widget (both the launcher and the widget window) re-appear after it was previously hidden (via the hide API) by calling the following API.
+
+```javascript
+HelpshiftWidget("show");
+```
+
+<Admonition type="info" title="Note">
+
+This API will restore the visibility of Helpshift widget after it was previously hidden. For example, if the widget was open before hideAPI was called, then show API will display the widget in the open state. This API retains the visibility behavior.
+
+</Admonition>
+
+## Visibility States
+
+<Image
+  src="/static/books/web-chat/wc-chat-widget-hide-show-api.png"
+  width="full"
+/>{" "}
+
+## Update Helpshift Widget Config {#update-helpshift-widget-config}
+
+If you want to update the `helpshiftWidgetConfig` object after your web page has loaded, you can do it by calling the following API
+
+```javascript
+HelpshiftWidget("updateHelpshiftWidgetConfig");
+```
+
+<Admonition type="info" title="Note">
+
+This API is best for use with single page web applications.
+
+</Admonition>
+
+Example: In a single page application, if you are retrieving user related data after login, to pass that updated user data to Helpshift widget, you should update the global helpshiftWidgetConfig object with the user data and call the above API.
+
+```javascript
+// Initial helpshift config object during page load
+helpshiftWidgetConfig = {
+    platformId: "<YOUR_PLATFORM_ID>",
+    domain: "<YOUR_DOMAIN>",
+    .
+    .
+};
+.
+.
+.
+// After user login, you can update the user data in config object.
+helpshiftWidgetConfig.userId = "<LOGGED_IN_USER_ID>";
+HelpshiftWidget("updateHelpshiftWidgetConfig");
+```
+
+<Admonition type="danger" title="Important">
+
+Make sure you do not overwrite the `helpshiftWidgetConfig` object when trying to update it.
+
+</Admonition>
+
+<Admonition type="info" title="Note">
+
+1. This API is applicable to all the `helpshiftWidgetConfig` object options, such as user related data, language, full privacy, widgetOptions, widgetType, etc.
+2. Once the conversation has started, calling this API will not update the user identity or update Custom Issue Fields for an ongoing conversation.
+3. This API will cause the widget to reload. If the user has opened the widget and this API is called, then the user will see the loading indicator, and the conversation of the updated user will be displayed.
```

Comment:
RE: ` the conversation of the updated user will be displayed.`
- We should get this checked with product as here we should say something like - and the widget will load with the updated user

Reply:
We can say: `widget chat conversation will load with the updated user`
Let's check this with mridul once

Reply:
Done